src/EncryptedPayload.ts

```
@@ -51,11 +51,11 @@ class EncryptedPayload {
     const jsonObject = EncryptedPayload.validateJson(json);
 
     // validate obligatory fields
-    if (!jsonObject.hasOwnProperty("ciphertext")) {
+    if (!jsonObject.hasOwnProperty('ciphertext')) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)